src/CryptoAxus.Application/Features/Artist/PatchArtistUserWalletAddress/Request/PatchArtistRequestExample.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace CryptoAxus.Application.Features.Artist.PatchArtistUserWalletAddress.Request;
+
+public class PatchArtistRequestExample : IExamplesProvider<PatchArtistRequest>
+{
+    public PatchArtistRequest GetExamples()
+    {
+        {
+            JsonPatchDocument<UpdateArtistDto> patchDocument = new JsonPatchDocument<UpdateArtistDto>();
+            patchDocument.Operations.Add(new Operation<UpdateArtistDto>
+            {
+                op = "replace",
+                path = "/userWalletAddress",
+                value = "test.userWalletAddress"
+            });
```

Comment:
Should this example only contain one field? We are not updating the username here?

Reply:
Resolved